kubernetes/fineract-server-deployment.yml

```diff
@@ -59,7 +60,27 @@ spec:
         image: apache/fineract:latest
         env:
         - name: JAVA_OPTS
```

Comment:
@xurror remove the `JAVA_OPTS` completely, see https://issues.apache.org/jira/browse/FINERACT-900 for why